addon/templates/components/polaris-display-text.hbs

```diff
@@ -1,4 +1,4 @@
-{{#let (element (or @htmlTag "p")) as |DisplayText|}}
+{{#let (element (or (or @htmlTag @tagName) "p")) as |DisplayText|}}
```

Comment:
Can just be

```suggestion
{{#let (element (or @htmlTag @tagName "p")) as |DisplayText|}}
```

instead of nesting `or`s? Same applies in a few other places where you've added `htmlTag` support.